src/components/doctor/DoctorPatients.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import {StyledPatientsList} from "./DoctorPatients.styled";
+// import EmptyList from "./EmptyList";
+import Patient from "./Patient";
+
+const DoctorPatients = () => {
+    return (
+        <StyledPatientsList>
+
+            {/*<EmptyList />*/}
```

Comment:
Remove

Reply:
> Remove

solved this question

Reply:
You can add this component when a condition was successful
For example: if your components state is an empty array you can render this component <EmptyList /> if it's not you render 
 <StyledPatientsList>

Reply:
> You can add this component when a condition was successful
> For example: if your components state is an empty array you can render this component if it's not you render

I will add .json file with data to render my patients . Thanks